### PR TITLE
Fix Ui issues and some documentation errors.

### LINKED ISSUE #100: [PE-D] Small Documentation Bug
May want to rearrange the order of features in the UG. List and Find feature is in between add, edit and delete features even thought add, edit and delete student features are stated to be in the same heading in the feature list

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: Justinnje/ped#3

### PR DESCRIPTION
**Pull request by bryans17**
Fixes #100, #102, #103, #105, #106, #108, #109, #110, #116.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -54,6 +54,31 @@ If you can type fast, TAB can get your Lab management tasks done faster than tra
...
+  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+
+* Items with `…` after them can be used multiple times including zero times.<br>
+  e.g. `[t/TAG]…` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+
+* Parameters can be in any order.<br>
+  e.g. if the command specifies `n/NAME g/GITHUB`, `g/GITHUB n/NAME` is also acceptable.
+
+* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
+  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
```
Comment (channne) [inline docs/UserGuide.md:75]: I think this `p/` has to be changed, because we deleted the `Phone` attribute

File: docs/UserGuide.md [hunk 2]
```diff
@@ -54,6 +54,31 @@ If you can type fast, TAB can get your Lab management tasks done faster than tra
...
+**:information_source: Notes about the command format:**<br>
+
+* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+
+* Items in square brackets are optional.<br>
+  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+
+* Items with `…` after them can be used multiple times including zero times.<br>
+  e.g. `[t/TAG]…` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
```
Comment (channne) [inline docs/UserGuide.md:69]: Perhaps we should change the pre-existing tag examples because it's a bit weird to tag students as `t/friend` and `t/family`

**Review by channne — COMMENTED** [review]
Just some minor things!

File: docs/UserGuide.md (hunk 1 above)
Comment (bryans17) [reply docs/UserGuide.md:75]: Thanks for the catch!

File: docs/UserGuide.md (hunk 2 above)
Comment (bryans17) [reply docs/UserGuide.md:69]: HAHA yea we should.

**Review by channne — APPROVED** [review]
LGTM 👍

**Pull request merged**